odoo/tests/common.py

```diff
@@ -843,7 +843,11 @@ def __init__(self, logger, window_size, test_class):
             'Page.frameStoppedLoading': self._handle_frame_stopped_loading,
             'Page.screencastFrame': self._handle_screencast_frame,
         }
-        self._receiver = threading.Thread(target=self._receive, name="WebSocket events consumer")
+        self._receiver = threading.Thread(
+            target=self._receive,
+            name="WebSocket events consumer",
+            args=(getattr(threading.current_thread(), 'dbname', None),)
```

Comment:
You could use get_db_name maybe. (Cleaner, more reliable even if it is a slight change of behaviour)

Reply:
Sure that works.